io.github.obsidian-level-maker.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.obsidian-level-maker
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: obsidian.sh
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --filesystem=home:rw
+modules:
+  - name: obsidian
+    buildsystem: cmake
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: http://github.com/obsidian-level-maker/Obsidian
+        tag: Obsidian-v20-20230509
+        commit: fb2c9f6269ced76643fa4c362738784ebae26cf7
+    build-commands:
+      - install -D obsidian /app/bin/obsidian
+  - name: startscript
```

Comment:
this should be merged with the main module.

Reply:
Why?

Reply:
because that's a good practice.

Reply:
To me, this is a logical separation of building the application itself and 'building' the extra shell script and files associated with being a flatpak.